climada/util/api_client.py

```diff
@@ -1071,4 +1194,27 @@ def rm_empty_dirs(directory: Path):
                 directory.rmdir()
             except OSError:  # raised when the directory is not empty
                 pass
+
         rm_empty_dirs(target_dir)
+
+    def get_dataset_file(self, **kwargs):
+        """Convenience method. Combines ``get_dataset`` and ``download_dataset``.
+        Returns the path to a single file if the dataset has only one,
+        otherwise throws an error.
+
+        Parameters
+        ----------
+        **kwargs
+            arguments for get_dataset and download_dataset
```

Comment:
```suggestion
            arguments for get_dataset and download_dataset
```
It is only `download_dataset` right?

Reply:
Yes. The ones for download_dataset are explicitly separated and all the others are submitted to get_dataset.